data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,2721 @@
+			`	"It is pretty much guaranteed there will be one such ship at all times around their home system, Ya-Hai, the center of their trade. For obvious reasons we cannot scan that ship ourselves, and we prefer that the Hai be ignorant of our interest in that design, anyway. Bring us the full scanner logs and you will be well rewarded."`
+			choice
+				`	"Seems easy. I'll do it."`
+					goto agree
+				`	"Why the sudden interest for that ship?"`
+					goto design
+				`	"You could just ask the Hai that come to you from their space about it."`
+			`	"That is what we will be forced to do if you refuse, but we would prefer this to be kept a secret. So do you agree to do this for us?"`
+			choice
+				`	"It seems easy enough. I'll do it."`
+					goto agree
+				`	"How do I scan it?"`
+					goto scan
+				`	"I don't want to get in trouble with the Hai."`
+			`	"That should not be a problem, the Hai will not bothered about you scanning their ship, if they even notice it."`
+			label design
+			`	"We would like to know how they avoided vulnerabilities in the hull of a ship of such size. Extra space on a ship cannot only be used for cargo... I will tell you more about it later, once you bring us the scans. That is, if you are willing of course."`
+			choice
+				`	"Sure, will do."`
+					goto agree
+				`	"How do I scan it?"`
+			label scan
+			`	"The information provided by one of our value detectors should be sufficient. If you don't have one, you can install one on Darkcloak, on the way there. You may want to get multiple scanners so you can scan the ship from a longer range, and faster. Please do actively scan it. Passive intel gathered on the fly will not do."`
+			choice
+				`	"Sure, will do."`
+					goto agree
+				`	"I'm not interested in scanning ships for you."`
+					decline
+			label agree
+			`	On the way back to your ship, Mera runs up to you. "One... last... thing, captain!" You let her catch her breath. "We have noticed one Geocoris just entered <waypoints> as expected, but seems to have some special modifications. Please scan that one specifically. We have marked its signature for you to recognize it. We want to have a detailed scan of its cargo. Also, I've just found a captain willing to give you one of our scanners. It will be placed in your cargo hold if there is no space on your ship."`
+				accept
+		outfit "Value Detector"
+		log "People" "Mera" `An Unfettered engineer who is interested in ship designs of all kinds. She is easily excited by new technologies and ships.`
+	on waypoint
+		dialog `You do not see the Geocoris that Tear-Fi mentioned; it's probably in a system nearby. You should land and depart when it arrives.`
```

Comment:
Its because missions dont do this kind of thing
Its a hack... Yes I have no way to make the ship enter as you enter the system otherwise, usually these are made staying which is dull.
The spawning of those ships is done when you land